packages/sdk-android/src/kotlinParser.ts

```diff
@@ -16,32 +16,32 @@ import { getBuildFilePath, getEntryFile, getAppId, addSystemInjects } from '@rnv
 //     androidwear: '"assets://index.androidwear.bundle"',
 // };
 
-export const parseFlipperSync = (c: Context, scheme: 'debug' | 'release') => {
-    const appFolder = getAppFolder(c);
-    const { platform } = c;
+// export const parseFlipperSync = (c: Context, scheme: 'debug' | 'release') => {
```

Comment:
@ElenaDiachenko please remove the commented out code

Reply:
@ElenaDiachenko why has this been commented out? are we not supporting flipper ?